test/test_synth_chorus_reverb.c

```diff
@@ -0,0 +1,39 @@
+
+#include "test.h"
+#include "fluidsynth.h"
+
+// this test should make sure that sample rate changed are handled correctly
+int main(void)
+{
+    fluid_synth_t *synth;
+    fluid_settings_t *settings = new_fluid_settings();
+    TEST_ASSERT(settings != NULL);
+    
+    TEST_SUCCESS(fluid_settings_setnum(settings, "synth.reverb.roomsize", 1.0));
+    TEST_SUCCESS(fluid_settings_setnum(settings, "synth.reverb.damp", 1.0));
+    TEST_SUCCESS(fluid_settings_setnum(settings, "synth.reverb.width", 1.0));
+    TEST_SUCCESS(fluid_settings_setnum(settings, "synth.reverb.level", 1.0));
+    
+    TEST_SUCCESS(fluid_settings_setint(settings, "synth.chorus.nr", 99));
+    TEST_SUCCESS(fluid_settings_setnum(settings, "synth.chorus.level", 1.0));
+    TEST_SUCCESS(fluid_settings_setnum(settings, "synth.chorus.speed", 1.0));
+    TEST_SUCCESS(fluid_settings_setnum(settings, "synth.chorus.depth", 1.0));
+  
+    synth = new_fluid_synth(settings);
+    TEST_ASSERT(synth != NULL);    
+    
+    TEST_ASSERT(fluid_synth_get_reverb_roomsize(synth) == 1.0);
+    TEST_ASSERT(fluid_synth_get_reverb_damp(synth) == 1.0);
+    TEST_ASSERT(fluid_synth_get_reverb_width(synth) == 1.0);
+    TEST_ASSERT(fluid_synth_get_reverb_level(synth) == 1.0);
+    
+    TEST_ASSERT(fluid_synth_get_chorus_nr(synth) == 99);
+    TEST_ASSERT(fluid_synth_get_chorus_level(synth) == 1.0);
+    TEST_ASSERT(fluid_synth_get_chorus_speed_Hz(synth) == 1.0);
```

Comment:
I just noticed that there is this capital "H" in ```fluid_synth_get_chorus_speed_Hz```. Maybe we should clean that up as well before the 2.0 release?

Reply:
Not sure. This function is already in use for 1.1.x for long. I dont like it as well, if you'd asked me I would simply call it `*_speed()`. However one might consider this as an unnecessary API change, motivated by personal taste.

Reply:
Alright, consistency is a good argument. Shall we call it `fluid_synth_get_chorus_speed()` then?